example_test.go

```diff
@@ -0,0 +1,170 @@
+package sqlair_test
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/canonical/sqlair"
+)
+
+func ExampleQuery_Get() {
+	type Employee struct {
+		ID   int    `db:"employee_id"`
+		Name string `db:"name"`
+		Team string `db:"team_name"`
+	}
```

Comment:
What do you think of moving the employee type outside so that we can reuse across the examples?